src/java/org/apache/cassandra/service/StartupChecks.java

```diff
@@ -285,6 +286,49 @@ public void execute()
         }
     };
 
+    public static final StartupCheck checkReadAheadKbSetting = new StartupCheck()
+    {
+        // This value is in KB.
+        private static final long EXPECTED_READ_AHEAD_KB_SETTING = 8;
```

Comment:
can we rename this to `MAX_RECOMMENDED_READ_AHEAD_KB_SETTING`, and set the max recommended to 128 (according to [this post](https://tobert.github.io/pages/als-cassandra-21-tuning-guide.html)) in case people use a slightly higher value for this ? We can keep the suggested value in the startup check error message to 8.

Reply:
Changed to 128, thanks